src/components/CollectionsPage/OptionsDropdown/OptionsDropdown.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Dropdown } from 'decentraland-ui'
+import classnames from 'classnames'
+import { preventDefault } from 'lib/preventDefault'
+import { Props } from './OptionsDropdown.types'
+import styles from './OptionsDropdown.module.css'
+
+const OptionsDropdown = (props: Props) => {
```

Comment:
Could this Component be re-used on the Scenes card of the root page?
![image](https://user-images.githubusercontent.com/692077/125851960-c2144af5-33d7-49f2-8b12-9569f987e732.png)

Reply:
Done!